src/components/Pool.vue

```diff
@@ -18,19 +18,19 @@
           <div class="pool-info">
             <token-logo :token-symbol="getAssetSymbol(liquidityItem.firstAddress)" size="small" />
             <div>{{ t('pool.pooledToken', { tokenSymbol: getAssetSymbol(liquidityItem.firstAddress) }) }}</div>
-            <div v-if="liquidityItem.firstBalance" class="pool-info-value">{{ liquidityItem.firstBalance }}</div>
+            <div class="pool-info-value">{{ getFirstBalance(liquidityItem) }}</div>
           </div>
           <div class="pool-info">
             <token-logo :token-symbol="getAssetSymbol(liquidityItem.secondAddress)" size="small" />
             <div>{{ t('pool.pooledToken', { tokenSymbol: getAssetSymbol(liquidityItem.secondAddress) }) }}</div>
-            <div v-if="liquidityItem.secondBalance" class="pool-info-value">{{ liquidityItem.secondBalance }}</div>
+            <div class="pool-info-value">{{ getSecondBalance(liquidityItem) }}</div>
           </div>
           <div class="pool-info">
             <pair-token-logo :first-token-symbol="getAssetSymbol(liquidityItem.firstAddress)" :second-token-symbol="getAssetSymbol(liquidityItem.secondAddress)" size="mini" />
             <div>{{ t('pool.pairTokens', { pair: getPairTitle(getAssetSymbol(liquidityItem.firstAddress), getAssetSymbol(liquidityItem.secondAddress)) }) }}</div>
-            <div class="pool-info-value">{{ liquidityItem.balance }}</div>
+            <div class="pool-info-value">{{ getBalance(liquidityItem) }}</div>
           </div>
-          <!-- TODO: uncomment when it will work -->
+          <!-- TODO: we decided to hide it because of many requests like getLiquidityReserves -->
           <!-- <div class="pool-info pool-info--share">
```

Comment:
Why did you change this class? I can't find any other parts with renaming (in styles, for example).

Reply:
I didn't change it...